virttest/storage.py

```diff
@@ -441,6 +441,19 @@ def get_backup_set(filename, backup_dir, action, good):
                 continue
             backup_func(src, dst)
 
+    def rm_backup_image(self):
+        """
+        Remove backup image
+        """
+        backup_dir = utils_misc.get_path(self.root_dir,
+                                         self.params.get("backup_dir", ""))
+        image_name = os.path.join(backup_dir, "%s.backup" % self.image_filename)
```

Comment:
I think it would be `os.path.basename(self.image_filename)`

Reply:
1. that would point to original image which is not the target to be deleted, it should point to .backup file.
2. we should be using `backup_dir`, it can be different than `images_base_dir`

Reply:
Sorry, let me clarify it: the `self.image_filename` part in this line should be changed to `os.path.basename(self.image_filename)`, please take the `backup_image` method as a reference.

Reply:
got it. Thanks.